applications/solvcon/prepare-solvcon-dev.sh

```diff
@@ -78,3 +85,11 @@ pushd ${SCSRC}/sandbox/gas/tube
 ./go run
 popd
 
+
```

Comment:
This workaround will be removed when the whole story of integration between `devenv` and `SOLVCON` is done.

Reply:
What is the time / schedule of the whole integration?  Is the workaround intended to be checked in?

Reply:
I could not tell the E.T.A. right now. Let's evaluate the schedule by:
1. Checking how many packages are needed for SOLVCON
2. Checking how many packages are able to be built by `devenv`

In my pull request, `gmsh` has been integrated. I believe most packages could be integrated in the same way. So a rough schedule may be:
1. 2 man-days to complete above package enumeration
2. 2 man-days implementation mocking how `gmsh` is integrated
3. 2 man-days testing

Maybe 1 man-week?

Reply:
`shunit2` looks promising to me https://github.com/kward/shunit2/ . I will have a look of it in more details.

Reply:
> I could not tell the E.T.A. right now. Let's evaluate the schedule by:
> 
> 1. Checking how many packages are needed for SOLVCON
> 2. Checking how many packages are able to be built by `devenv`
> 
> In my pull request, `gmsh` has been integrated. I believe most packages could be integrated in the same way. So a rough schedule may be:
> 
> 1. 2 man-days to complete above package enumeration
> 2. 2 man-days implementation mocking how `gmsh` is integrated
> 3. 2 man-days testing
> 
> Maybe 1 man-week?

A rough estimate like weeks or months suffices for me.